go.mod

```diff
@@ -1,25 +1,28 @@
 module github.com/adobe/kratos
 
-go 1.14
+go 1.15
```

Comment:
Why not 1.16 ?

Reply:
I used version listed in operator Prerequisites https://sdk.operatorframework.io/docs/installation/

but I don't see any reasons for not using 1.16

Reply:
Updated to 1.16